data/bingo/functions/tick/handle_player_join.mcfunction

```diff
@@ -10,7 +10,8 @@ function bingo:util/find_player_team
 
 # Resource pack is likely not loaded here. In case it isn't, just print
 # hardcoded English message.
-execute if score $game_state bingo.state matches 3 run tellraw @s[predicate=bingo:is_in_game] {"translate": "%1074992263$s%1$s", "with": [["", {"text": "Click here", "color": "#00c3ff", "clickEvent": {"action": "run_command", "value": "/trigger bingo.lobby"}}, " to go back to the lobby\n", {"text": "Click here", "color": "#00c3ff", "clickEvent": {"action": "run_command", "value": "/trigger bingo.spectator"}}, " to switch to spectator mode."], [{"translate": "bingo.game.go_to_lobby", "with": [{"translate": "bingo.game.go_to_lobby.click_lobby", "color": "#00c3ff", "clickEvent": {"action": "run_command", "value": "/trigger bingo.lobby"}}, {"translate": "bingo.game.go_to_lobby.click_spectator", "color": "#00c3ff", "clickEvent": {"action": "run_command", "value": "/trigger bingo.spectator"}}]}]]}
+execute if score $game_state bingo.state matches 3 run tellraw @s[predicate=bingo:is_in_game] {"translate": "%1074992263$s%1$s", "with": [["", {"text": "Click here", "color": "#00c3ff", "clickEvent": {"action": "run_command", "value": "/trigger bingo.lobby"}}, " to go back to the lobby\n", {"text": "Click here", "color": "#00c3ff", "clickEvent": {"action": "run_command", "value": "/trigger bingo.spectator"}}, " to switch to spectator mode."], [{"translate": "bingo.game.menu.options", "with": [{"translate": "bingo.game.menu.options.click_lobby", "color": "#00c3ff", "clickEvent": {"action": "run_command", "value": "/trigger bingo.lobby"}}, {"translate": "bingo.game.menu.options.click_spectator", "color": "#00c3ff", "clickEvent": {"action": "run_command", "value": "/trigger bingo.spectator"}}]}]]}
+execute if score $game_state bingo.state matches 3 run tellraw @s[predicate=bingo:is_in_game] {"translate": "%1074992263$s%1$s", "with": [["", {"text": "Hint: You can click ", "color": "gray", "italic": true}, {"text":"[", "color":"white"}, {"text":"≡", "color":"#00c3ff", "clickEvent": {"action": "run_command", "value": "/trigger bingo.menu"}, "hoverEvent":{"action":"show_text","contents":["Bingo Menu"]}}, {"text":"]", "color":"white"}, {"text": " or use the command ", "color": "gray", "italic": true}, {"text": "/trigger bingo.menu", "color": "white", "clickEvent": {"action": "suggest_command", "value": "/trigger bingo.menu"}}, {"text":" at any time to bring up the bingo menu.", "color":"gray", "italic":true}], [{"translate": "bingo.game.menu.hint", "color": "gray", "italic": true, "with": [{"translate": "bingo.game.menu.text", "color": "white", "italic": false, "with": [{"translate": "bingo.game.menu.icon", "color": "#00c3ff", "italic": false, "clickEvent": {"action": "run_command", "value": "/trigger bingo.menu"}, "hoverEvent":{"action":"show_text","contents":[{"translate":"bingo.game.menu.hover_text"}]}}]}, {"text": "/trigger bingo.menu", "color": "white", "italic": false, "clickEvent": {"action": "suggest_command", "value": "/trigger bingo.menu"}}]}]]}
```

Comment:
Maybe use these commands in the util functions instead, so print_menu can be called here?

Reply:
I've now moved those tellraws to the relevant print functions and called [those](https://github.com/NeunEinser/bingo/blob/10c5ff0ce89b9fd3d8302485baa4f538633836b0/data/bingo/functions/tick/handle_player_join.mcfunction#L13) instead. 
I added the `execute as @s[predicate=bingo:is_in_game]` just before calling the function, although perhaps that predicate check could be moved to within the function itself?
Also, you'd note that I kept them as two separate tellraws for now since here we want to print the `options` and the `menu_hint`, whereas the `print_menu` function prints the `options` and the `options_hint`. So then if you want to skip the second "execute if", should I create a separate print function, that prints this combination: `options` and `menu_hint`?

The other option you suggested was merging the two tellraw commands into one long tellraw command, although it is quite unreadable:
```mcfunction
execute if score $game_state bingo.state matches 3 run tellraw @s[predicate=bingo:is_in_game] {"translate": "%1074992263$s%1$s", "with": [["", {"text": "Click here", "color": "#00c3ff", "clickEvent": {"action": "run_command", "value": "/trigger bingo.lobby"}}, " to go back to the lobby\n", {"text": "Click here", "color": "#00c3ff", "clickEvent": {"action": "run_command", "value": "/trigger bingo.spectator"}}, " to switch to spectator mode.", "\n", {"text": "Hint: You can click ", "color": "gray", "italic": true}, {"translate":"[%s]", "color":"white", "italic":false, "with":[{"text":"≡", "color":"#00c3ff", "clickEvent":{"action":"run_command", "value":"/trigger bingo.menu"}, "hoverEvent":{"action":"show_text", "contents":[{"translate":"bingo.game.menu.hover_text"}]}}]}, {"text": " or use the command ", "color": "gray", "italic": true}, {"text": "/trigger bingo.menu", "color": "white", "clickEvent": {"action": "suggest_command", "value": "/trigger bingo.menu"}}, {"text":" at any time to bring up the bingo menu.", "color":"gray", "italic":true}], [{"translate": "bingo.game.menu.options", "with": [{"translate": "bingo.game.menu.options.click_lobby", "color": "#00c3ff", "clickEvent": {"action": "run_command", "value": "/trigger bingo.lobby"}}, {"translate": "bingo.game.menu.options.click_spectator", "color": "#00c3ff", "clickEvent": {"action": "run_command", "value": "/trigger bingo.spectator"}}]}, "\n", {"translate": "bingo.game.menu.hint", "color": "gray", "italic": true, "with": [{"translate":"[%s]", "color":"white", "italic":false, "with":[{"text":"≡", "color":"#00c3ff", "clickEvent":{"action":"run_command", "value":"/trigger bingo.menu"}, "hoverEvent":{"action":"show_text", "contents":[{"translate":"bingo.game.menu.hover_text"}]}}]}, {"text": "/trigger bingo.menu", "color": "white", "italic": false, "clickEvent": {"action": "suggest_command", "value": "/trigger bingo.menu"}}]}]]}
```

Reply:
Both is fine